R/aldex.R

```diff
@@ -2,7 +2,7 @@
 ##'
 ##' 
 ##' @title ALDEx3 Linear Modles
-##' @param counts an (D x N) matrix of sequence count, N is number of samples, D
```

Comment:
this change does not have to be made now. Its superficial. I want to minimize changes when reviewing more important changes.